tool/tbot/config/configtemplate_ssh.go

```diff
@@ -40,6 +44,66 @@ type TemplateSSHClient struct {
 	ProxyPort uint16 `yaml:"proxy_port"`
 }
 
+// openSSHVersionRegex is a regex used to parse OpenSSH version strings.
+var openSSHVersionRegex = regexp.MustCompile(`^OpenSSH_(?P<major>\d+)\.(?P<minor>\d+)(?:p(?P<patch>\d+))?`)
+
+// openSSHMinVersionForRSAWorkaround is the OpenSSH version after which the
+// RSA deprecation workaround should be added to generated ssh_config.
+var openSSHMinVersionForRSAWorkaround = semver.New("8.5.0")
+
+// parseSSHVersion attempts to parse
+func parseSSHVersion(versionString string) (*semver.Version, error) {
+	versionTokens := strings.Split(versionString, " ")
+	if len(versionTokens) == 0 {
+		return nil, trace.Errorf("invalid version string: %s", versionString)
```

Comment:
nit: prefer BadParameter to Errorf? 

```suggestion
		return nil, trace.BadParameter("invalid version string: %s", versionString)
```

Same for others.